pkg/avx/elfdump.go

```diff
@@ -0,0 +1,84 @@
+// +build ignore
+
+/*
+Copyright 2020 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"encoding/hex"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+	"text/template"
+)
+
+const (
+	blocksPerRow = 12
+)
+
+type Program struct {
+	ProgramLines []string
+}
+
+func main() {
+	f, err := ioutil.ReadFile("../../libexec/avx512.o")
```

Comment:
Maybe use the binary directory instead of working directory?

Reply:
the eBPF ELF is generated to `libexec` directory. Are you asking to change the `Makefile` to output to `bin` directory?

Reply:
No, I just meant that the file path is now relative to the working directory, meaning that you can run this binary only from the same directory where the binary is located. I was thinking that it should maybe be relative to the directory where the binary is. But this is more of a matter of taste I think, so no worries.